js/src/forum/components/TextEditor.js

```diff
@@ -22,25 +23,22 @@ export default class TextEditor extends Component {
     super.oninit(vnode);
 
     /**
-     * The value of the textarea.
+     * The value of the editor.
      *
      * @type {String}
      */
     this.value = this.attrs.value || '';
+
+    /**
+     * Whether the editor is disabled.
+     */
+    this.disabled = !!this.attrs.disabled || true;
```

Comment:
What is this? Won't `this.disabled` always equal true here? If `this.attrs.disabled` is falsy, you will still be setting it to true.